easybuild/easyblocks/p/petsc.py

```diff
@@ -275,8 +279,9 @@ def configure_step(self):
                 self.cfg.update('configopts', ' '.join([withdep + spec for spec in ['=1', inc_spec, lib_spec]]))
 
             # set PETSC_DIR for configure (env) and build_step
-            env.setvar('PETSC_DIR', self.cfg['start_dir'])
-            self.cfg.update('buildopts', 'PETSC_DIR=%s' % self.cfg['start_dir'])
+            petsc_dir = os.path.abspath(self.cfg['start_dir'])
```

Comment:
If this is really only about the trailing slash, maybe we should use `.rstrip(os.path.sep)` instead?

```python
>>> import os
>>> os.path.sep
'/'
>>> path_with_slash = '/tmp/example/'
>>> path_with_slash.rstrip(os.path.sep)
'/tmp/example'
```

Using `abspath` will resolve potential symbolic links , which we should avoid I think...

Reply:
Yes, the problem is only the trailing slash. So .rstrip is better.